src/components/MoneyTransaction.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import styles from './MoneyTransaction.module.css'
+import { action } from '@storybook/addon-actions'
+import { Button } from './Button'
+import { Table } from './Table'
+import { CreateEntry } from './Form'
+import { Heading, HeadingDisabled } from './Text'
+
+export const MoneyTransaction = () => {
+  return (
+        <div className = {`${styles.page}`}>
```

Comment:
The template literal is not required here

```suggestion
        <div className={styles.page}>
```